.github/workflows/build-and-push.yml

```
@@ -0,0 +1,81 @@
+name: Build and Push
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  buildx:
+    name: Build and Push the Docker Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Compute Tag
+        uses: actions/github-script@v6
+        id: compute-tag
+        with:
+          result-encoding: string
+          script: |
+            if (context.eventName === "pull_request") {
+              return "pr-" + context.issue.number;
+            }
+            if (context.eventName === "push") {
+              if (context.ref.startsWith("refs/tags/")) {
+                return context.ref.slice(10);
+              }
+              if (context.ref === "refs/heads/develop") {
+                return "latest";
+              }
+            }
+            return "FALSE";
+
+      - uses: actions/checkout@v2
+
+      -
+        name: Docker Login
+        id: docker-login
+        env:
+          DOCKER_USER: ${{secrets.DOCKER_USERNAME}}
+          DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+        run: |
+          docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+
+      -
+        name: Install Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@master
+        with:
+          version: latest
+
+      -
+        name: Build the Image
+        env:
+          TAG: ${{ steps.compute-tag.outputs.result }}
+        run: |
+          docker buildx build --push \
+          --tag ${{secrets.DOCKER_USERNAME}}/artemis-maven-template:$TAG \
+          --platform linux/amd64,linux/arm64/v8 .
```

```suggestion
          --platform linux/amd64,linux/arm64/v8 .
         if: ${{ steps.compute-tag.outputs.result != 'FALSE' }}         
```
(Not sure if the indentation is correct in the suggestion - please double check)